onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/ontology/data/handler/maturity/OntologyHandler.java

```diff
@@ -22,27 +22,28 @@
  * @author Patrycja Miazek (patrycja.miazek@makolab.com)
  */
 @Service
-public class FiboOntologyHandler {
+public class OntologyHandler {
 
     private static final String ONTOLOGY_IRI_GROUP_NAME = "ontologyIri";
     private static final Pattern ONTOLOGY_IRI_PATTERN
             = Pattern.compile("(?<ontologyIri>.*\\/)[^/]+$");
     private static final String PROD = "prod";
-    private static final String PROD_DEV = "dev";
-    private static final String DEV  = "prodDev";
-    private static final String PROD_ICON  = "release";
-    private static final String PROD_DEV_ICON  = "mixed";
-    private static final String DEV_ICON  = "develop";
+    private static final String PROD_DEV_MIXED = "prodDev";
```

Comment:
In https://github.com/edmcouncil/onto-viewer/pull/215#discussion_r857642579 I asked for a change to 'prod_and_dev_mixed'.